packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+## 2.4.2 - Aug 23 2021
+
+Incremental version for Teams Toolkit with multiple bugs fixed and the following updates：
+
+New Feature:
+
+- Enable create project from quick start page.
+- Enable report issue from local debug error pop-up message.
+- Add CLI command 'teamsfx preview' to preview SPFx apps.
```

Comment:
This has already been added in the previous changelog version 2.4.1?

Reply:
Thank you for careful checking!  yes this is a mistake. I have deleted it.